examples/values/backend-org-1.yaml

```diff
@@ -125,6 +125,13 @@ orchestrator:
     - mychannel:
         restricted: false
         model_export_enabled: true
+        chaincode:
+          name: mycc
```

Comment:
We don't have a distributed mode, why to add these values?

Reply:
Side-effect of the rebase (distributed mode was still a thing at the time I created those files). Removing it, thank you.